code/datums/diseases/advance/symptoms/symptoms.dm

```diff
@@ -10,7 +10,7 @@
 	var/stage_speed = 0
 	var/transmittable = 0
 	// The type level of the symptom. Higher is harder to generate.
-	var/level = 0
+	var/level = -1

```

Comment:
Since level 0 is now being used as a real symptom level, I don't want the symptom that wasn't supposed to be accessible to be accessible with the anomolous virus food